src/nupic/algorithms/Connections.hpp

```diff
@@ -332,6 +332,15 @@ namespace nupic
         SynapseData dataForSynapse(const Synapse& synapse) const;
 
         /**
+         * Returns the synapses for the source cell that they synapse on.
+         *
+         * @param presynapticCell(int) Source cell index
+         *
+         * @return (set)Synapse indices
+         */
+        std::vector<Synapse> synapsesForPresynapticCell(const Cell& presynapticCell) const;
```

Comment:
This is not necessary; see comment https://github.com/numenta/nupic.core/pull/409/files#r34511536.